internal/core/config.go

```diff
@@ -45,6 +45,7 @@ type Configuration struct {
 	GolangciLint   GolangciLintConfiguration    `yaml:"golangciLint"`
 	SpellCheck     SpellCheckConfiguration      `yaml:"spellCheck"`
 	GitHubWorkflow *GithubWorkflowConfiguration `yaml:"githubWorkflow"`
+	Renovate       RenovateWorkflowConfig       `yaml:"renovate"`
```

Comment:
I missed this earlier. The struct should not have workflow in its name as it is not a workflow.